py/runai_model_streamer/runai_model_streamer/tests/test_annotations.py

```
@@ -0,0 +1,85 @@
+        """Import every module in the package. Returns (modules, skipped)."""
+        modules = []
+        skipped = []
+        for module_info in pkgutil.walk_packages(
+            runai_model_streamer.__path__, prefix=f"{runai_model_streamer.__name__}."
+        ):
+            name = module_info.name
+            if ".tests" in name:
+                continue
+            try:
+                modules.append(importlib.import_module(name))
+            except Exception as error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)